scripts/extensions/pool_update.precheck

```diff
@@ -149,6 +156,33 @@ def execute_precheck(session, control_package, yum_conf_file, update_precheck_fi
             else:
                 raise PrecheckFailure(errmsg)
         else:
+            regex = ERROR_XML_START + '.+' + ERROR_XML_END
+            m = re.search(regex, output, flags=re.DOTALL)
```

Comment:
I think I'd rather see a new the evaluation of the XML done in a separate function and make it data driven to make extending it in the future easier.